avm/src/daemon/ctrl.rs

```diff
@@ -902,4 +970,165 @@ impl ControlPort {
     let bytes = hyper::body::to_bytes(res.body_mut()).await?;
     Ok(std::str::from_utf8(&bytes)? == "true")
   }
+
+  pub async fn dskeys(self: &ControlPort, ip: &str) -> (String, Vec<String>) {
+    (
+      ip.to_string(),
+      self.dskeys_inner(ip).await.unwrap_or(Vec::new()),
+    )
+  }
+
+  async fn dskeys_inner(self: &ControlPort, ip: &str) -> DaemonResult<Vec<String>> {
+    let url = format!("https://{}:4142/datastore/keys", ip);
+    let req = Request::builder().method("GET").uri(url);
+    let cluster_secret = CLUSTER_SECRET.get().unwrap().clone().unwrap();
```

Comment:
seeing so many of `CLUSTER_SECRET.get().unwrap().clone().unwrap()` makes me think maybe we should have convenience getters for our `OnceCell`/`Lazy` variables